translations/fr.json

```diff
@@ -1,61 +1,67 @@
+  "markOrderFulfillStatus": "Demande de test terminée",
+  "newStatus": "NOUVEAU",
+  "nextPage": "Page suivante",
+  "noLabRequestsFoundCheckFilters": "Aucune demande d'analyse trouvée. Veuillez vérifier vos filtres et réessayer.",
+  "onHoldStatus": "EN_ATTENTE",
+  "onOrAfterDateFilter": "Filtrer les demandes à partir de :",
+  "option": "Choisissez une option",
+  "orderer": "Prescripteur",
+  "orderNumber": "Numéro de la demande",
+  "orderPickedSuccessfully": "Vous avez sélectionné une demande avec succès",
+  "orders": "Demandes",
   "patient": "Patient",
-  "pickedAnOrder": "Picked an order",
-  "pickRequest": "Pick Lab Request",
-  "pickRequestConfirmationText": "Continuing will update the request status to \"In Progress\" and advance it to the next stage. Are you sure you want to proceed?",
-  "pickupLabRequest": "Pickup Lab Request",
-  "previousPage": "Previous page",
-  "procedure": "procedure",
-  "receivedStatus": "RECEIVED",
-  "reject": "Reject",
-  "rejectLabRequest": "Reject Lab Request",
-  "rejectLabRequestSuccessMessage": "",
-  "rejectLabRequestTitle": "Lab Request Rejected",
-  "results": "Results",
-  "save": "Save",
-  "searchThisList": "Search this list",
-  "status": "Status",
-  "tabletOverlay": "Tablet overlay",
-  "testOrderCompletedSuccessfully": "You have successfully completed the test order",
-  "testsOrdered": "Tests ordered",
-  "testType": "Test Type",
-  "totalorders": "Total Orders",
-  "updateEncounter": "Update lab results",
-  "urgency": "Urgency",
-  "view": "View ",
-  "worklist": "Ex-Worklist"
+  "pickedAnOrder": "Demande sélectionnée",
+  "pickRequest": "Sélectionner la demande de laboratoire",
+  "pickRequestConfirmationText": "Continuer mettra à jour le statut de la demande à \"En cours\" et la fera avancer à l'étape suivante. Êtes-vous sûr de vouloir continuer?",
+  "pickupLabRequest": "Sélectionner la demande",
+  "previousPage": "Page précédente",
+  "procedure": "Procédure",
+  "receivedStatus": "REÇU",
+  "reject": "Rejeter",
+  "rejectLabRequest": "Rejeter la demande",
+  "rejectLabRequestSuccessMessage": "Demande d'analyse rejetée avec succès",
+  "rejectLabRequestTitle": "Demande d'analyse rejetée",
+  "results": "Résultats",
+  "save": "Enregistrer",
+  "searchThisList": "Rechercher dans la liste",
+  "status": "Statut",
+  "tabletOverlay": "Prise de plusieurs médicaments",
+  "testOrderCompletedSuccessfully": "Vous avez terminé la demande de test avec succès",
+  "testsOrdered": "Demandes d'analyses",
+  "testType": "Type de test",
+  "totalorders": "Total des demandes",
+  "updateEncounter": "Mettre à jour les résultats d'analyse",
+  "urgency": "Urgence",
+  "view": "Voir",
+  "worklist": "Analyses en cours",
+  "noLabRequestsFoundC": "Aucune demande d'analyse trouvée",
+  "checkFilters": "Veuillez vérifier les filtres ci-dessus et réessayer",
+  "startDate":"Date de début",
+  "endDate":"Date de fin",
+  "totalOrders":"Total des demandes",
+  "itemsPerPage": "Éléments par page"
```

Comment:
Thank you for working on this, @azizdiakite! So, we don't modify non-`en` translation files by hand. Instead, we use the Transifex integration for that. These strings ought to be translated via the Transifex UI. Automated PRs will then provide the updated strings back to the repo ([similar to this one](https://github.com/openmrs/openmrs-esm-laboratory-app/pull/98)).

Read more about this [here](https://om.rs/translate).